js/content/store.js

```diff
@@ -2852,22 +2850,12 @@ let WishlistPageClass = (function(){
         ExtensionLayer.runInPageContext(() =>
             $J(document).ajaxSuccess(function( event, xhr, settings ) {
                 if (settings.url.endsWith("/remove/")) {
-                    window.postMessage({
-                        type: "es_remove_wl_entry",
-                        removed_wl_entry: settings.data.match(/(?!appid=)\d+/)[0]
-                    }, "*");
+                    Messenger.sendMessage("removeWlEntry", settings.data.match(/(?!appid=)\d+/)[0]);
```

Comment:
The message text can be anything, and can include spaces. I'm a fan of "As explicit and readable as possible", so "remove-wishlist-entry" or "remove wishlist entry" or anything without an abbreviation all seem better than "removeWlEntry".

Reply:
I'd go for "removeWishlistEntry" here just to keep in style with the other messages